samples/net/wifi/sample.yaml

```diff
@@ -9,4 +9,7 @@ tests:
     platform_whitelist: cc3220sf_launchxl disco_l475_iot1
   sample.net.wifi.esp_8266:
     extra_args: SHIELD=esp_8266
-    platform_whitelist: frdm_k64f sam4e_xpro disco_l475_iot1
+    platform_whitelist: sam4e_xpro
+  sample.net.wifi.esp_8266_arduino:
+    extra_args: SHIELD=esp_8266_arduino
+    platform_whitelist: frdm_k64f disco_l475_iot1
```

Comment:
Shouldn't this commit be squashed with previous one, which removes `arduino_serial` from `frdm_k64f.overlay`? To preseve bisectability.

Reply:
done.